recipes/comlrl/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "1.2.6" %}
+
+package:
+  name: comlrl
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/OpenMLRL/CoMLRL.git
+  git_rev: v{{ version }}
```

Comment:
Please use a tarball archive and a hash/checksum. GitHub tags are not immutable.